code/modules/power/port_gen.dm

```diff
@@ -217,6 +217,10 @@
 /obj/machinery/power/port_gen/pacman/attack_paw(mob/user)
 	interact(user)
 
+/obj/machinery/power/port_gen/pacman/CtrlClick(mob/user)
+	if(anchored)
+		TogglePower()
```

Comment:
```suggestion
	if(!user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
		return
	if(anchored)
		TogglePower()
```
Assuming you don't want TK to work on this still.

Reply:
Why not, but still, add a topic check